packages/driver/cypress/integration/commands/log_spec.js

```diff
@@ -0,0 +1,495 @@
+      if (log.get('name') === 'nextUntil') {
+        cy.removeAllListeners('log:added')
+        expect(log.get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('div').nextUntil(document.getElementsByClassName('warning')[0])
+  })
+
+  it('parentsUntil - jquery', () => {
+    cy.on('log:added', (attrs, log) => {
+      if (log.get('name') === 'parentsUntil') {
+        cy.removeAllListeners('log:added')
+        expect(log.get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('div').parentsUntil(cy.$$('.warning'))
+  })
+
+  it('parentsUntil - DOM', () => {
+    cy.on('log:added', (attrs, log) => {
+      if (log.get('name') === 'parentsUntil') {
+        cy.removeAllListeners('log:added')
+        expect(log.get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('div').parentsUntil(document.getElementsByClassName('warning')[0])
+  })
+
+  it('prevUntil - jquery', () => {
+    cy.on('log:added', (attrs, log) => {
+      if (log.get('name') === 'prevUntil') {
+        cy.removeAllListeners('log:added')
+        expect(log.get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('div').prevUntil(cy.$$('.warning'))
+  })
+
+  it('prevUntil - DOM', () => {
+    cy.on('log:added', (attrs, log) => {
+      if (log.get('name') === 'prevUntil') {
+        cy.removeAllListeners('log:added')
+        expect(log.get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('div').prevUntil(document.getElementsByClassName('warning')[0])
+  })
+
+  it('select', () => {
+    cy.on('log:added', (attrs) => {
+      if (attrs.name === 'select') {
+        expect(this.logs[1].get('options')).to.deep.eq({})
+      }
+    })
+
+    cy.get('#auto-test').select(['Cypress', 'Jest'])
+  })
+
+  it('spread', () => {
+    cy.on('fail', () => {
+      expect(this.logs[1].get('options')).to.deep.eq({
+        timeout: 1000,
+      })
+    })
+
+    cy.wrap([1, 2, 3]).spread({ timeout: 1000 }, (a, b, c) => {
+      throw new Error()
+    })
+  })
+
+  it('task', () => {
+    cy.on('fail', () => {
+      expect(this.logs[0].get('options')).to.deep.eq({})
+    })
+
+    cy.task('throw', { a: 1, b: 2 })
+  })
+
+  it('then', () => {
+    cy.on('fail', () => {
+      expect(this.logs[1].get('options')).to.deep.eq({
+        timeout: 1000,
+      })
+    })
+
+    cy.wrap('a').then({ timeout: 1000 }, () => {
+      throw new Error('hi?')
+    })
+  })
+
+  it('uncheck', () => {
+    cy.on('fail', () => {
+      expect(this.logs[1].get('options')).to.deep.eq(null)
+    })
+
+    cy.get('#b').uncheck(['us', 'ca'])
+  })
+})
+
+const log = require('../../../src/cypress/log')
+
+describe('display props are relayed correctly', () => {
+  it('timeout and wallClockStartedAt', () => {
+    const props = log.getDisplayProps({
+      timeout: 4000,
+      wallClockStartedAt: new Date().toJSON(),
+    })
+
+    expect(props).to.haveOwnProperty('timeout')
+    expect(props).to.haveOwnProperty('wallClockStartedAt')
+  })
+})
```

Comment:
I tried to add an e2e test for `timeout` and `wallClockStartedAt`. But failed. The log display props are filtered at https://github.com/cypress-io/cypress/blob/develop/packages/runner/src/lib/event-manager.js#L285 and `'reporter:log:add'` event cannot be used in the tests.

Reply:
It would probably have to be a test on the driver just making sure that the log object includes the correct properties. However, that's most likely outside of the scope of this PR and I can work on that separately.